packages/nhost-js/src/utils/helpers.ts

```diff
@@ -25,7 +26,7 @@ export function urlFromSubdomain(
     throw new Error('Either `backendUrl` or `subdomain` must be set.')
   }
 
-  // check if subdomain is [http[s]://]localhost[:port]
+  // check if subdomain is [http[s]://]localhost[:port] or [http[s]://]localdev[:port]
   const subdomainLocalhostFound = subdomain.match(LOCALHOST_REGEX)
   if (subdomainLocalhostFound?.groups) {
     const { protocol = 'http', host, port = 1337 } = subdomainLocalhostFound.groups
```

Comment:
```suggestion
    const { protocol = 'http', host, port } = subdomainLocalhostFound.groups
```

goes with the following suggestion